WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,21 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read(Rails.root.to_s + "/app/views/regulations/wca-regulations.json")).freeze
+  rescue
+    REGULATIONS = [].freeze
```

Comment:
Ahh, darn. For the aforementioned reasons about failing fast vs failing silently, I think I would prefer to see Travis build the regulations, unless that's a painful operation (but it shouldn't be too painful anymore, should it?)

If building the regulations is painful, what about a cute hate where travis does a `echo blah > WcaOnRails/app/views/regulations/wca-regulations.json` to simulate building the regulations before running tests?